msg/RadarTrack.msg

```diff
@@ -0,0 +1,12 @@
+# All variables below are relative to the radar's frame of reference.
+# This message is not meant to be used alone but as part of a stamped or array message.
+
+uint16 track_id                             # The track ID of the object generated by the radar.
```

Comment:
Since the name of the message is "RadarTrack", would it be simpler/intuitive to use `id` instead of `track_id`?

Reply:
I like the use of the unique_msgs/UUID and just calling it a uuid since that's clear

Reply:
Does UUID imply a certain format though?  Not sure if all radars would provide the data in the UUID format, or is that something the drivers would have to handle?

Reply:
This is just an id generator tool, so anything should be able to generate a UUID. Its more or less just a long string of numbers like a big uint16 but its a standard way of dealing with data like this

Reply:
My preference is for ID over UUID. UUID's are 128 bits long (uint8[16] as per http://docs.ros.org/api/uuid_msgs/html/msg/UniqueID.html). UUID seems like a complete overkill for a numbering system that will seldomly go above 100 and usually a lot less. UUID also implies numbers will be universally unique. I suspect most drivers will output an integer ID and the numbering will reset when the module does. ID at least does not imply the numbers are universal unique

Reply:
This conversation was marked resolved, but it seems like it was resolved the wrong way? Sounds like we agreed to use a simple integer ID but a UUID was committed.

To clarify the discussion in this thread. I believe @radarAaron meant it would seldomly go above 100 *for a single frame*, meaning the radar is rarely tracking more that 100 objects at one instant in time. From my limited automotive radar experience, tracked objects are usually given an 8-bit integer ID.
I'm not against UUID, it's definitely handy, but it's not something that any radar hardware (that I've used) would provide. I think for the majority of radar hardware, we'd have to sort of fill in this UUID from the ROS driver side.

Reply:
I haven't considered this conversation as resolved (although it has been marked so, maybe automatically by a commit changing the line?). 

Again, there seems to be some confusion which might have been removed if there were a better description of what this message actually is and should be. @icolwell-as is talking about IDs unique among all tracked objects in a single time instant, while @SteveMacenski is talking about IDs aggregated over days of operation.

Reply:
I had understood that UUID was better from an ROS/scalability/multi-sensor perspective and committed on that basis. I agree with @icolwell-as that we would have to fill this out driver-side as the hardware would generally would supply an 8bit ID, but that is not too difficult to do.